requirements.txt

```diff
@@ -1,2 +1 @@
 Flask
-mysql-connector-python
```

Comment:
Hi @SaarCohen93,
Probably we should not delete mysql-connector-python from requirements.txt. Looks like a regression.

Reply:
Fixed.

Reply:
Not fixed yet